s3tests/functional/test_s3.py

```diff
@@ -185,6 +185,7 @@ def _get_alt_connection():
 @attr(method='get')
 @attr(operation='list all keys')
 @attr(assertion='pagination w/max_keys=2, no marker')
+@attr('skip_for_storj') # todo: needs lexicographically ordered objects
```

Comment:
Might be worth adding [storj/gateway-mt#109](https://github.com/storj/gateway-st/issues/49) to the TODO.

Reply:
Done!